.ruby-version

```diff
@@ -0,0 +1 @@
+2.4.5
```

Comment:
technically this is not needed as it is already in `test/Gemfile` (also versions are different)

If you want  you can remove it from there and leave this `.ruby_version`. They are two different ways to do the same thing.

Reply:
The Gemfile doesn't seem to cantain any references to the ruby version:
```
# frozen_string_literal: true
source "https://rubygems.org"

gem 'monster', :git => 'git@bitbucket.org:onfido/monster.git', :tag => '0.8.26'
gem 'cucumber', :git => 'https://github.com/cucumber/cucumber-ruby.git', :ref => '085be64'
gem 'cucumber-api', :git => 'https://github.com/cristian-m-vasile/cucumber-api.git'
gem 'rake'
gem 'selenium-webdriver'
gem 'browserstack-local'
```

The `Gemfile.lock` has the version of bundler used but not the ruby version either. Am I missing something?

Reply:
This Gemfile will be soon 🔥 as I'm replacing the last Ruby UI test.

Reply:
cool I guess no point for this then